server/migrations/20170608140411-create-roles.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
```

Comment:
**Code Organization - Nitpick**
 
- Why you have `migrations` and `models` in duplicate?